manifests/3/360/360zip/4.0.0.1340/360.360zip.locale.zh-CN.yaml

```diff
@@ -0,0 +1,27 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: 360.360zip
+PackageVersion: 4.0.0.1340
+PackageLocale: zh-CN
+Publisher: 360安全中心
+PublisherUrl: https://www.360.cn
+PublisherSupportUrl: https://bbs.360.cn/forum.php?mod=forumdisplay&fid=120
+PrivacyUrl: http://www.360.cn/privacy/v3/360yasuo.html
+#Author: 
+PackageName: 360压缩
+Moniker: 360zip
```

Comment:
```suggestion
```
![image](https://user-images.githubusercontent.com/11600822/130212516-8617a677-9096-44a7-a886-2f9cdcf7383a.png)
keep the one that's in en-US.